crates/aptos/src/move_tool/lint.rs

```diff
@@ -0,0 +1,65 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use super::IncludedArtifactsArgs;
+use crate::common::types::{CliCommand, CliError, CliTypedResult, MovePackageDir};
+use aptos_framework::{BuildOptions, BuiltPackage};
+use async_trait::async_trait;
+use clap::Parser;
+use move_compiler_v2::Experiment;
+use move_model::metadata::CompilerVersion;
+
+/// Run a Lint tool to show additional warnings, in addition to ordinary warnings and/or errors
+/// generated by the Move 2 compiler.  Currently this is done by running the Move 2 compiler with a
+/// special pass, so some other compiler options are also available, but in the future this may be a
+/// dedicated tool.
+///
+/// Ideally, we would skip code generation, but currently this is not possible. We do omit
+/// metadata generation, which hopefully renders the output not able to be uploaded.
+#[derive(Parser)]
```

Comment:
When I run `aptos move lint --help`, all of this text shows up. While all of this might be true, it is very disruptive and confusing for the user to read.

Reply:
Removed.